target/scripts/helpers/aliases.sh

```diff
@@ -30,6 +30,17 @@ function _handle_postfix_regexp_config() {
   fi
 }
 
+function _handle_postfix_regexp_send_only_config() {
+  : >/etc/postfix/regexp-send-only
```

Comment:
I'd prefer not creating a file explicitly when it's not used.

Reply:
I was just copying the regular postfix-regexp logic here — would I be OK to also remove the same thing in the `_handle_postfix_regexp_config()` above (and corresponding `# TODO: Investigate why this file is always created, nothing seems to append only the cp below?` comment)?

Reply:
There's probably a related issue already open about details for that.

Without looking for that, I can't quite say. I've not had as much time for DMS as I'd like, but if the change was delayed I may have detailed why. I'd leave it alone for now, unless you have time to sink 😅 

Proper actionable feedback from me is going to need to wait until I have the time to refresh on related context like this. I'll try to get that sorted for you by this weekend if possible, sorry about the friction that adds.